velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -17,7 +17,7 @@
 #pragma once
 
 #include "velox/core/Expressions.h"
-#include "velox/substrait/SubstraitUtils.h"
+#include "velox/substrait/SubstraitParser.h"
```

Comment:
Thank you for renaming. It would be nice submit this rename as a separate PR. Smaller PRs are easier to review.

Reply:
Submitted a new PR for the renaming: https://github.com/facebookincubator/velox/pull/1591.